src/inference/src/dev/core_impl.cpp

```diff
@@ -1095,8 +1095,9 @@ std::shared_ptr<const ov::Model> ov::CoreImpl::apply_auto_batching(const std::sh
             const auto disabled = batch_mode->second.as<std::string>() == CONFIG_VALUE(NO);
             // virtual plugins like AUTO/MULTI will need the config
             // e.g. to deduce the #requests correctly
+            // proxy plugin should also keep the config
             // otherwise, no need for this config key in the rest of loading
-            if (!is_virtual_device(deviceName))
+            if (!is_virtual_device(deviceName) && !is_proxy_device(deviceName))
```

Comment:
For proxy plugin we have a check in the line 1109.

Could you please add new tests? 
Why do we need these changes?

Reply:
does this mean with proxy enabled, currently GPU will not enable auto batch (no matter which batch config applied)?
e.g user compiles to GPU with throughput hint, no auto batching will be checked/applied if proxy is there?

Reply:
Why do you think that GPU doesn't work with auto batch?
Right now all tests on GPU with auto batch are passed.
We add auto batch between proxy and GPU:
Proxy->Batch->GPU. It should work for the master branch

Reply:
sorry, let me share the case more clearly:
considering below case：
core.compile_model("GPU", ov::hint::performance_mode("THROUGHPUT"), **ov::allow_auto_batching(false)**)；
above expect batch to be turned off
yet on current master with proxy plugin enabled, it's not the case
you can try below command: ./benchmark_app -m ~/work/mount/mobilenet-v2/onnx/onnx/FP16/INT8/mobilenet-v2.xml -d GPU -b 1
here -b 1 will add ov::allow_auto_batching(false) to compile property
but still get batch compiled model:
[ INFO ]   NETWORK_NAME: torch-jit-export
[ INFO ]   SUPPORTED_CONFIG_KEYS: AUTO_BATCH_TIMEOUT
[ INFO ]   EXECUTION_DEVICES: OCL_GPU.0
[ INFO ]   AUTO_BATCH_TIMEOUT: 1000

Reply:
@songbell Could you add a test on this behaviour?

Reply:
update tests